collections/DOrville/MS_DOrville_153.xml

```diff
@@ -37,45 +40,132 @@
+                        <note>Bk. 2, fol. 29v; bk. 3, fol. 58r; bk. 4, fol. 93v; bk. 5, fol. 118r.</note>
+                        <note type="gloss">With selective but sometimes extensive interlinear glossing, and accompanied by a selective commentary, written both in the margin and on inserted sheets.</note>
+                        <note>Fol. 47, which is apparently part of the original volume (the end of quire 3) and not an inserted sheet, was originally blank, with a form(?) letter and other notes added on fol. 47v and commentary added on fol. 47r.</note>
+                     </msItem>
+                     <msItem>
+                        <title type="desc" key="work_16389">Commentary on Boethius, De consolatione philosophiae</title>
+                        <incipit><locus>(fol. 7r)</locus> <quote>Quisquis composito serenus euo fatum sub pedibus</quote> Istud est quartum metrum huius primi libri quod vocatur <sic>volenticum</sic> ab inuentore, tropheicum a pede predominate, pentametrum a numero pedum <gap reason="editorial"/> In quo metro Philosophia exortans Boecium ad virtutum constantiam tria facit  </incipit>
+                        <incipit><locus>(fol. 29v)</locus> Secundus liber Boecii. Iste est 2us liber Boecii de consolatu philosophie in quo postquam in primo libro inuestigauit morbum Boecii diligenter inquirendo   </incipit>
+                        <incipit><locus>(fol. 58r)</locus> Iste est tertius liber Boecii de consolacione philosophiae sic continuatur ad precedentes </incipit>
+                        <note>No commentary on book 5.</note>
+                        <!-- not Trevet or Wheatley (there are some resemblances to Wheatley) -->
+                     </msItem>
+                     
+                  </msItem>
+                  <msItem>
+                     <locus>(fol. 133r–v)</locus>
+                     <note>Added inscriptions and pen-trials etc., including:</note>
+                     <msItem>
+                        <note>(fol. 133r)</note>
+                        <note>‘Ve t<ex>ibi</ex> cu nigre dicebat cacubus olle’; ‘Aristotoles(?) Appetitus divitiarum <sic>cressit</sic>
+                           ad infinitum’; ‘Qui scit frenare linguam sensumque domare fortior est e(?) qui vincit  viribus orbes’; ‘in quibus estis qui(?) estis’; ‘Morbos multos multa fercula fecerunt’ (Seneca, Epistulae morales); ‘Si hec duo pronomina meum et tuum tollentur(?) <gap reason="editorial"/>’</note></msItem>
+                     <msItem>
+                        <locus>(fol. 133v)</locus>
+                        <note>Pen-trials etc., including the names ‘Johan’, ‘Willem’(?) and ‘Hoest’, and grammatical notes.</note>
+                     </msItem>
                   </msItem>
                </msContents>
                <physDesc>
+                  <p><secFol>extulisset ipsum</secFol></p>
```

Comment:
I have tended to make `<secFol>` a child of `<collation>`, since that seems to be the one place it is allowed, but I'm not sure if that is actually desirable.

Reply:
It's a problem! Is is physical description? Is is provenance?